rbac/management/querysets.py

```diff
@@ -81,29 +82,31 @@ def get_group_queryset(request):
     """Obtain the queryset for groups."""
     scope = request.query_params.get(SCOPE_KEY, ACCOUNT_SCOPE)
     if scope != ACCOUNT_SCOPE:
-        return get_object_principal_queryset(request, scope, Group)
+        return filter_queryset_by_tenant(get_object_principal_queryset(request, scope, Group), request.tenant)
```

Comment:
I wonder if putting this filter_queryset_by_tenant within the get_object_principal_queryset could save some efforts.

Looks there are some lint complains again, maybe worth running tox-e lint locally

Reply:
Gave that a shot, we'll see what the linter thinks this time